shared/engine/index.platform.native.js

```diff
@@ -51,15 +49,12 @@ class NativeTransport extends TransportShared {
 function createClient (incomingRPCCallback: incomingRPCCallbackType, connectCallback: connectDisconnectCB, disconnectCallback: connectDisconnectCB) {
   const client = sharedCreateClient(new NativeTransport(incomingRPCCallback, connectCallback, disconnectCallback))
 
+  nativeBridge.start()
```

Comment:
We kick off the other side, this is what RN wants to avoid races between the JS loading and native loading